### PR TITLE
updates imageswipe widget

### PR DESCRIPTION
**Pull request by sevresbabylone**
## Image Widget
- Updates Imageswipe widget. Demo up [here](https://swipes-for-science-imageswipe.netlify.app/#/)
- Updates ImageSwipe seed data
- Adds Imageswipe to Tutorial, follows the same format as WordSwipe

## database.rules.json
- votes: Adds 'userId' as index to votes to query all votes with userId
- games/$gameId: Adds rule to allow writes only if gameId doesn't exist, or gameId exists and belongs to the currentUser

## Home Page
- Play button text switches to Resume Game if user already has a game in progress (assume user will always have a game in progress once starting a game)

## Profile Page
- Updates Average duration (which was a placeholder value) in Profile page calculated from all votes from currentUser
- Adds Pause screen, start duration for the sample is reset when game is unpaused
- Adds logout to Profile screen

## Configuration files
- Neatens configuration files for example wordswipe and image swipe, this will be cleaned up further when we update the docs
- Eg. needsTutorial shifted under tutorial

## removes assessment mode
- Reverted back to data annotation mode, taking 0 as the left value and 1 as right value (Check Play.vue in component archive)

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ router.beforeEach((to, from, next) => {
   } else if (to.name === "Play" && currentUser) {
     get(child(dbRef, `/users/${currentUser.uid}`)).then(snapshot => {
       const userData = snapshot.val();
-      if (!userData.taken_tutorial && config.needsTutorial) {
+      if (!userData.taken_tutorial && config.tutorial.needsTutorial) {
```
Comment (kylesome) [inline src/router/index.js:146]: just wondering if `taken_tutorial` is meant to be camelCase?

File: src/components/Widgets/ImageSwipe/Card.vue [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+      const x = this.position.x + event.delta.x;
+      const y = this.position.y + event.delta.y;
+      let rotation = 0;
+      if (this.position.x > 0) {
+        rotation = Math.min(MAX_ROTATION, this.position.x * 2);
+      } else {
+        rotation = Math.max(-MAX_ROTATION, this.position.x * 2);
+      }
+
+      this.setPosition({ x: x, y: y, rotation: rotation });
```
Comment (kylesome) [inline src/components/Widgets/ImageSwipe/Card.vue:95]: could just use `{ x, y, rotation }` here if you wanted

**Review by kylesome — COMMENTED** [review]
Looks great! 👍

File: src/router/index.js (hunk 1 above)
Comment (sevresbabylone) [reply src/router/index.js:146]: i followed the previous naming, but it should be updated. it has to be updated in a few places so i'll address it in another PR.

**Pull request merged**